M2/Macaulay2/packages/Complexes/ChainComplexDoc.m2

```diff
@@ -2017,25 +2017,24 @@ doc ///
    SeeAlso
      (dual, Complex)
      (prune, Complex)
+     (homology, ZZ, Complex)
 ///
 
 
 
 doc ///
    Key
      (homology,ZZ,Complex)
-     (cohomology,ZZ,Complex)
    Headline
-     homology or cohomology module of a complex
+     homology of a complex
```

Comment:
`HH^i C` was removed because we want to use this for complexes of sheaves, and it wasn't doing anything deep here, just `HH_-i C` which the user can do as well.